Product/Production/Services/DocumentSubmissionCore/src/main/java/gov/hhs/fha/nhinc/docsubmission/outbound/deferred/response/StandardOutboundDocSubmissionDeferredResponse.java

```diff
@@ -156,26 +155,24 @@ private XDRAcknowledgementType createFailedPolicyCheckResponse() {
         return response;
     }
 
-    private void auditRequestFromAdapter(
-            RespondingGatewayProvideAndRegisterDocumentSetSecuredResponseRequestType request, AssertionType assertion) {
-        auditLogger.auditEntityXDRResponseRequest(request, assertion, NhincConstants.AUDIT_LOG_INBOUND_DIRECTION);
-    }
-
-    private void auditResponseToAdapter(XDRAcknowledgementType response, AssertionType assertion) {
-        auditLogger.auditEntityAcknowledgement(response, assertion, NhincConstants.AUDIT_LOG_OUTBOUND_DIRECTION,
-                NhincConstants.XDR_RESPONSE_ACTION);
+    private void auditRequest(RegistryResponseType request, AssertionType assertion,
+        NhinTargetCommunitiesType targets) {
+        auditLogger.auditRequestMessage(request, assertion,
+            MessageGeneratorUtils.getInstance().convertFirstToNhinTargetSystemType(targets),
+            NhincConstants.AUDIT_LOG_OUTBOUND_DIRECTION, NhincConstants.AUDIT_LOG_NHIN_INTERFACE, Boolean.TRUE, null,
+            NhincConstants.NHINC_XDR_RESPONSE_SERVICE_NAME);
     }
 
     protected boolean hasNhinTargetHomeCommunityId(
-            RespondingGatewayProvideAndRegisterDocumentSetSecuredResponseRequestType request) {
+        RespondingGatewayProvideAndRegisterDocumentSetSecuredResponseRequestType request) {
 
         if (request != null
-                && request.getNhinTargetCommunities() != null
-                && NullChecker.isNotNullish(request.getNhinTargetCommunities().getNhinTargetCommunity())
-                && request.getNhinTargetCommunities().getNhinTargetCommunity().get(0) != null
-                && request.getNhinTargetCommunities().getNhinTargetCommunity().get(0).getHomeCommunity() != null
-                && NullChecker.isNotNullish(request.getNhinTargetCommunities().getNhinTargetCommunity().get(0)
-                        .getHomeCommunity().getHomeCommunityId())) {
+            && request.getNhinTargetCommunities() != null
+            && NullChecker.isNotNullish(request.getNhinTargetCommunities().getNhinTargetCommunity())
+            && request.getNhinTargetCommunities().getNhinTargetCommunity().get(0) != null
+            && request.getNhinTargetCommunities().getNhinTargetCommunity().get(0).getHomeCommunity() != null
+            && NullChecker.isNotNullish(request.getNhinTargetCommunities().getNhinTargetCommunity().get(0)
```

Comment:
This if check is redundant here. We can simply do a return (request != null
                && request.getNhinTargetCommunities() != null
                && NullChecker.isNotNullish(request.getNhinTargetCommunities().getNhinTargetCommunity())
                && request.getNhinTargetCommunities().getNhinTargetCommunity().get(0) != null
                && request.getNhinTargetCommunities().getNhinTargetCommunity().get(0).getHomeCommunity() != null
                && NullChecker.isNotNullish(request.getNhinTargetCommunities().getNhinTargetCommunity().get(0)
                        .getHomeCommunity().getHomeCommunityId()));